frontends/PyCDE/test/tst_matrix.py

```diff
@@ -0,0 +1,150 @@
+# RUN: py-split-input-file %s
```

Comment:
nit: i think the convention for file naming for pytest (and maybe other common testing frameworks) is to spell out `test_`: https://docs.pytest.org/en/7.1.x/explanation/goodpractices.html#conventions-for-python-test-discovery. We don't always do that here, but it might be nice

Reply:
Will do - there's not much of a convention in place at the moment for PyCDE test files. I plan to do a passthrough here soon to clean up things and create some form of convention.